overlays/tracing-overlay.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# Tracing overlay
+#
+# This overlay enhances cos-lite with the distributed tracing backend Tempo.
+applications:
+  tempo:
+    charm: tempo-k8s
+    scale: 1
+    trust: true
+    channel: edge
+
+relations:
+- [tempo:ingress, traefik:traefik-route]
+# self-monitoring relations to cos-lite
+- [tempo:logging, loki:logging]
+- [grafana:grafana-dashboard, tempo:grafana-dashboard]
+- [grafana:grafana-source, tempo:grafana-source]
+- [prometheus:metrics-endpoint, tempo:metrics-endpoint]
+# tracing relations (monitoring cos-lite)
+- [traefik:tracing, tempo:tracing]
+- [prometheus:tracing, tempo:tracing]
+- [grafana:tracing, tempo:tracing]
+- [loki:tracing, tempo:tracing]
+- [alertmanager:tracing, tempo:tracing]
+- [catalogue:tracing, tempo:tracing]
```

Comment:
TLS complicates things: would need a separate overlay probably? No need at this point?
Same for the offers and 3-node overlays.

Should we have tempo overlays for cos under the tempo repo?

Reply:
What would putting the tempo overlays in the tempo repo solve? we still have a combinatorics issue for the tls, offers, and 3-node overlays. Is it a matter of tidiness, then we could categorize them. 

```
/overlays
  /3-node
    /cos-lite
    /tracing
  /tls
    /cos-lite
    /tracing
  /offers
    /cos-lite
    /tracing
```

Reply:
but I think bundle and overlays should live together

Reply:
btw there's no 3-node overlay yet as tempo as it stands can't be scaled. We're working on the scaling-monolithic mode.

Reply:
ended up putting all under /overlays/tracing

Reply:
ended up putting all under /overlays/tracing